src/app/components/login/login.component.ts

```diff
@@ -16,18 +17,30 @@ import { Router } from '@angular/router';
   templateUrl: './login.component.html',
   styleUrl: './login.component.css'
 })
-export class LoginComponent {
+export class LoginComponent implements OnInit {
   passwordFormControl = new FormControl('', [Validators.required, Validators.minLength(8)]);
   emailFormControl = new FormControl('', [Validators.required, Validators.pattern(Regex.Email)]);
   loginError: string | null = null;
 
   email: string = '';
   password: string = '';
+  user: any = null;
 
-  constructor(private router: Router){
+  constructor(private router: Router, private authService: AuthService){
      
   }
 
+  async ngOnInit() {
+    try {
+      this.user = await this.authService.getCurrentUser();
+      if (this.user) {
+        this.router.navigate(['/board/1']);  
```

Comment:
please talk to @GerniVisser-bbd and find out what the default navigation should be

Reply:
He said board/1 is fine for now @Adrian-Hawkins